x-pack/plugins/infra/public/components/asset_details/tabs/overview/tile.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useMemo } from 'react';
+
+import { i18n } from '@kbn/i18n';
+import {
+  EuiIcon,
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiI18n,
+  EuiToolTip,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import type { Action } from '@kbn/ui-actions-plugin/public';
+import { useLensAttributes } from '../../../../hooks/use_lens_attributes';
+import type { HostsLensMetricChartFormulas } from '../../../../common/visualizations';
+import { LensWrapper } from '../../../../pages/metrics/hosts/components/chart/lens_wrapper';
+import { buildCombinedHostsFilter } from '../../../../pages/metrics/hosts/utils';
+import { TooltipContent } from '../../../../pages/metrics/hosts/components/metric_explanation/tooltip_content';
```

Comment:
We're importing this from `../../../../pages/metrics/hosts`. Perhaps this should be moved to a common folder